### PR TITLE
Fix typo follow height: FLW_TGT_HT

### PR DESCRIPTION
**Pull request by hamishwillee**
@junwoo091400 This appears to have been renamed. Can you confirm?

### CODE REVIEW

File: src/modules/flight_mode_manager/tasks/AutoFollowTarget/follow_target_params.c [hunk 1]
```diff
@@ -95,7 +95,7 @@ PARAM_DEFINE_FLOAT(FLW_TGT_FA, 180.0f);
  *
  * Maintain altitude or track target's altitude. When maintaining the altitude,
  * the drone can crash into terrain when the target moves uphill. When tracking
- * the target's altitude, the follow altitude NAV_MIN_FT_HT should be high enough
+ * the target's altitude, the follow altitude FLW_TGT_HT should be high enough
```
Comment (hamishwillee) [inline src/modules/flight_mode_manager/tasks/AutoFollowTarget/follow_target_params.c:98]: @julianoes The only change I made was this one, through the github interface. Then all these other updates in other files appeared. Have you any idea what might have caused this? Do I need to be concerned?

**Comment by junwoo091400** [comment]
Thanks for the find! Yes this PR is correct. Regarding the ekf2 change indicator commit, that is probably from the Github Actions (I assume). Maybe the branch is outdated from the `main`? Could you rebase?

**Comment by junwoo091400** [comment]
I think this recent commit to main branch caused this diff. Could you rebase? (That would drop this auto commit in the PR as well as it will be empty)

https://github.com/PX4/PX4-Autopilot/commit/9b0b0d1168307ec2e2c40fd8e63e66439da05533

**Comment by hamishwillee** [comment]
Done. Though I think it is crap a rebase was forced upon me by that autocomit.

**Review by junwoo091400 — APPROVED** [review]

**Pull request merged**